src/send-discord-notification.ts

```diff
@@ -8,14 +8,13 @@ export default async function sendDiscordNotification(
   timelockId: string,
   txHash: string,
 ) {
-  const { title, description } = JSON.parse(event.args.description) as {
+  const { title } = JSON.parse(event.args.description) as {
     title: string;
-    description: string;
   };
 
   const message = new EmbedBuilder()
     .setTitle(title)
-    .setDescription(description)
+    .setDescription("Please review the full proposal in governance.mento.org")
```

Comment:
Can we use Markdown links here? Would be nice if we could link directly to the proposal instead of the gov UI landing page